components/quizSingle/styles.ts

```diff
@@ -280,21 +280,6 @@ export const TileNumber = styled.span`
   right: 15px;
 `;
 
-export const GraphSVG = styled.svg`
```

Comment:
(Related to first comment)
The style file _already exists_. Is there a specific reason to move styles out from here?